default/stringtables/en.txt

```diff
@@ -3478,6 +3478,116 @@ Double-click on an available item to add it to the production queue.
+Unavailable items, if displayed, will detail the reasons they are unavailable at the bottom of their tooltip in red (hover the mouse over the item).
+
+<u>Production Queue panel</u>
+
+(default position: left edge, below summary)
+The Production Queue panel contains Production Orders previously enqueued.
+Production Points are allotted to the Production Orders starting at the top of the queue.
+Any orders for the currently selected system will highlight the system name with your empires color (top right of the order).
+These orders may be rearranged in the queue by dragging and dropping to a new position.
+Right clicking on an order will show a menu for moving, removing, or pausing an order (as well as other possible actions, such as <i>Show in Pedia</i> ).
+When removing an order any previous progress on it is lost, there is no credit returned or given to other orders.
+Pausing an order prevents any progress on it until the order is resumed.
+For ships, an additional menu item is available to Rally to a system.  Once the order completes, the ship(s) will have a waypoint assigned for the system rallied to.  To set or change a Rally destination, select the desired system on the map, right click the Production Order, and select <i>Rally to </i>... (where ... is the system name).
+
+
+<u>Production Order</u>
+
+A Production Order is created through the Producible Items panel and is placed into the Production Queue panel.
+Each order defines a repetition count (1⟳) and a quantity (1x), selectable in the top left of the order.
+The repetition count will repeat the order after it completes, without adjusting the position in the queue.
+The quantity defines how many of the item will be produced each repetition.
+
+<i>Note</i>  Repetitions will only consume enough PP for the current iteration, unless the order will complete next turn.
+If the order will complete, some PP are allowed to start the next iteration, up to the maximum allowed per turn for that item type.
+
+
+<u>Example</u>
+
+Your empire produces 10 PP per turn.
+A Survey Ship costs 36 PP with a minimum of 3 turns (max 12 PP/turn).
+(For brevity, this fictional Survey Ship does not increase [[encyclopedia FLEET_UPKEEP_TITLE]])
+-Turn  1: You queue an order for Survey Ship and set the repetition count to 2. (order #1)
+                You queue another order for Survey Ship with a repetition of 2. (order #2)
+                You queue another order for Survey Ship and set the quantity to 2.
+
+-Turn  2: Order #1 has completed 10 PP, 26 PP remain.  10 PP is projected for next turn.
+
+    (Completed PP / Remaining PP to complete -> Projected next turn)
+
+-Turn  3: Order #1 20 / 16 -> 10
+
+-Turn  4: Order #1 30 /   6 ->   6  an additional 4 PP will go to the next repetition of this order.
+
+-Turn  5: A Survey Ship is produced (order #1)
+                Order #1   4 / 32 -> 10  now displays 1 repetition remains
+
+-Turn  6: Order #1 14 / 22 -> 10
+
+-Turn  7: Order #1 32 /   4 ->   4
```

Comment:
I come a bit late, but why the completed PP value raise from 14 to 32 in one turn? Although the projected PP value is only 10?